### PR TITLE
Update fachhochschule-vorarlberg-note.csl

### PR DESCRIPTION
**Pull request by cookie365**
added prefix and suffix
added URL, DOI, date should always be displayed in every type, if it is set
updated the style to to be equal with Update fachhochschule-vorarlberg-author-date.csl #2573

### CODE REVIEW

**Comment by damnation333** [comment]
Why did you make a new PR? why didn't you edit the other one. Makes it easier to trace back the history of the style when you have to go back. Also is just easier for the reviewers.

**Comment by cookie365** [comment]
Thanks for the information. I am not quite sure. Is this not a different style than fachhochschule-vorarlberg-author-date.csl? May I close this pull request so it gets merged into master?

**Comment by adam3smith** [comment]
Unless I'm overlooking something @damnation333 just didn't see these were different styles. I was a little confused by your commit message, too. As they are, let's keep both open, please. I'll try to review&merge as soon as possible.

**Comment by damnation333** [comment]
oh dear. my fault. didn't notice this is for the note style. sorry about the confusion.

**Comment by cookie365** [comment]
@damnation333 and @adam3smith sorry about the confusion, ok i will keep them open.

File: fachhochschule-vorarlberg-note.csl [hunk 1]
```diff
@@ -28,38 +28,54 @@
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
   <locale xml:lang="de">
+    <style-options punctuation-in-quote="true"/>
     <terms>
       <term name="et-al">u.a.</term>
-      <term name="anonymous" form="short">o.&#160;A.</term>
-      <term name="no date" form="short">o.&#160;J.</term>
+      <term name="anonymous" form="short">o. A.</term>
+      <term name="no date" form="short">o. J.</term>
```
Comment (rmzelle) [inline fachhochschule-vorarlberg-note.csl]: @cookie365, sorry for the long wait, but I have a question: it looks like you replaced all these escaped non-breaking spaces (`&#160;`) with regular spaces throughout the style. Was that intentional?

Comment (cookie365) [inline fachhochschule-vorarlberg-note.csl]: @rmzelle yes it was intentional, because the other Fachhochschule Vorarlberg Style also replaced them.

**Comment by rmzelle** [comment]
Okay, thanks!

**Pull request merged**